addons/im_livechat/models/chatbot_script_step.py

```diff
@@ -367,6 +368,25 @@ def _process_step_forward_operator(self, discuss_channel):
 
         return posted_message
 
+    def _to_store(self, store: Store, /, *, fields=None):
+        if fields is None:
+            fields = {
+                "answers": None,
+                "message": [],
+                "type": True,
+            }
+        for step in self:
+            data = self._read_format(
+                [f for f in fields if f not in {"answers", "message", "type"}], load=False
+            )[0]
+            if "answers" in fields:
+                data["answers"] = Store.many(step.answer_ids, fields=fields["answers"])
```

Comment:
Should be renamed `answer_ids` in JS (eventually).

Key in `fields` can already be renamed though.